lib/config/definitions.ts

```diff
@@ -1284,6 +1284,14 @@ const options: RenovateOptions[] = [
     allowedValues: ['branch', 'pr', 'pr-comment'],
     default: 'pr',
   },
+  {
+    name: 'automergeStrategy',
+    description:
+      'The merge strategy to use when automerging PRs. Used only if `automergeType=pr`',
```

Comment:
```suggestion
      'The merge strategy to use when automerging PRs. Used only if `automergeType=pr`.',
```

I think the style here is that we use a full stop at the end of the sentence. 😉